### DISCUSSION TITLE
Undo/Uninstall support for all commands implemented by this utility.

### DISCUSSION THREAD

**Discussion by ArchKaine** [ChrisTitusTech/linutil#368] [Ideas]
It would be nice to be able to test things that this utility does, without having to be permanently stuck with the changes. It would also be nice to have documentation of what each change does before enabling the function. IE, if the user activates a function, show the user a list of the changes that will be made, and explain what each change is. The user can then choose to implement that set of changes, or cancel them.

**Comment by HamonTeacher** [comment]
I agree, it is not even explained to me what this parameter is or how it will be changed. The program just starts installing right away. This is horrible, considering that apparently pure fedora doesn't have some fonts that are used by this utility and I get this horror ![image](https://github.com/user-attachments/assets/a4433241-d97f-4090-ac2d-86cca8c89c89).

**Comment by koibtw** [comment]
#384

**Comment by koibtw** [comment]
About your fonts, @HamonTeacher. What command (menu entry in linutil) exactly are you talking about? I will add needed fonts as deps (will install automatically when running the command).
